controllers/platformnetwork_controller_test.go

```diff
@@ -7,173 +7,171 @@ import (
+// 			crd_fetched := &starlingxv1.HostProfile{}
+
+// 			Eventually(func() bool {
+// 				err := k8sClient.Get(ctx, crd_key, crd_fetched)
+// 				if err == nil {
+// 					crd_fetched.ObjectMeta.Finalizers = []string{}
+// 					err = k8sClient.Update(ctx, crd_fetched)
+// 					if err == nil {
+// 						err = k8sClient.Delete(ctx, crd_fetched)
+// 						if err == nil {
+// 							return true
+// 						}
+// 					}
+// 				}
+// 				return false
+// 			}, timeout, interval).Should(BeTrue())
+
+// 			Eventually(func() bool {
+// 				err := k8sClient.Get(ctx, crd_key, crd_fetched)
+// 				return err != nil
+// 			}, timeout, interval).Should(BeTrue())
+
+// 		} else {
+// 			crd_fetched := &starlingxv1.Host{}
+
+// 			Eventually(func() bool {
+// 				err := k8sClient.Get(ctx, crd_key, crd_fetched)
+// 				if err == nil {
+// 					crd_fetched.ObjectMeta.Finalizers = []string{}
+// 					err = k8sClient.Update(ctx, crd_fetched)
+// 					if err == nil {
+// 						err = k8sClient.Delete(ctx, crd_fetched)
+// 						if err == nil {
+// 							return true
+// 						}
+// 					}
+// 				}
+// 				return false
+// 			}, timeout, interval).Should(BeTrue())
+
+// 			Eventually(func() bool {
+// 				err := k8sClient.Get(ctx, crd_key, crd_fetched)
+// 				return err != nil
+// 			}, timeout, interval).Should(BeTrue())
+// 		}
+// 	}
+// }
+
+// func DeletePlatformNetwork(nwk_name string) {
+// 	key := types.NamespacedName{
+// 		Name:      nwk_name,
+// 		Namespace: TestNamespace,
+// 	}
+
+// 	fetched := &starlingxv1.PlatformNetwork{}
+
+// 	Eventually(func() bool {
+// 		err := k8sClient.Get(ctx, key, fetched)
+// 		if err == nil {
+// 			fetched.ObjectMeta.Finalizers = []string{}
+// 			err = k8sClient.Update(ctx, fetched)
+// 			if err == nil {
+// 				err = k8sClient.Delete(ctx, fetched)
+// 				if err == nil {
+// 					return true
+// 				}
+// 			}
+// 		}
+// 		return false
+// 	}, timeout, interval).Should(BeTrue())
+
+// 	Eventually(func() bool {
+// 		err := k8sClient.Get(ctx, key, fetched)
+// 		return err != nil
+// 	}, timeout, interval).Should(BeTrue())
+// }
+
+// func SimulateVIMStrategyAction(hostname string, expect_strategy string) {
+// 	host_key := types.NamespacedName{
+// 		Name:      hostname,
+// 		Namespace: TestNamespace,
+// 	}
+// 	host_instance := &starlingxv1.Host{}
+
+// 	Eventually(func() bool {
+// 		err := k8sClient.Get(ctx, host_key, host_instance)
+// 		Expect(err).To(BeNil())
+// 		return (host_instance.Status.StrategyRequired == expect_strategy)
+// 	}, timeout, interval).Should(BeTrue())
+
+// 	if host_instance.Status.StrategyRequired == cloudManager.StrategyLockRequired {
+// 		HostsListBodyResponse = strings.Replace(HostsListBody, `"administrative": "unlocked",`, `"administrative": "locked",`, 1)
+// 		HostsListBodyResponse = strings.Replace(HostsListBodyResponse, `"operational": "enabled"`, `"operational": "disabled"`, 1)
+// 	} else {
+// 		HostsListBodyResponse = HostsListBody
+// 	}
+// }
```

Comment:
Please remove commented part.
Request to create a single test case where we just create the `PlatformNetwork` resource.
Expect `Reconciled = InSync = false`